email_alert_service/models/email_alert_template.rb

```diff
@@ -1,6 +1,11 @@
+require  "gds_api/content_store"
+
+
 class EmailAlertTemplate
   def initialize(document)
     @document = document
+    @content_store = GdsApi::ContentStore.new(Plek.new.find('content-store'))
```

Comment:
Other projects now have a `Services` module with references to external services, like this one:

https://github.com/alphagov/email-alert-api/blob/master/lib/services.rb

It would probably be good to use that in this repo too.